AutoTenez.py

```diff
@@ -17,17 +18,18 @@
 only_retrieve_your_external_reference = False # Set to True to retrieve your external reference to share with a friend
 dryrun = False # Only check available time slots, but don't make a reservation. False by default
 
+# Variables to be (optionally) filled in by input arguments
```

Comment:
`player2_external_reference` is absolutely not optional if you intend to make a reservation.